server/gc/src/test/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogsTest.java

```diff
@@ -287,13 +328,23 @@ public void replicationDelaysFileCollection() throws Exception {
     EasyMock.expectLastCall().once();
     EasyMock.expect(mscanner.iterator()).andReturn(replicationWork.entrySet().iterator());
     EasyMock.replay(context, fs, marker, tserverSet, rscanner, mscanner);
-    GarbageCollectWriteAheadLogs gc = new GarbageCollectWriteAheadLogs(context, fs, false,
-        tserverSet, marker, tabletOnServer1List) {
-      @Override
-      protected Map<UUID,Path> getSortedWALogs() {
-        return Collections.emptyMap();
-      }
-    };
+    GarbageCollectWriteAheadLogs gc =
+        new GarbageCollectWriteAheadLogs(context, fs, tserverSet, false) {
+          @Override
+          protected Map<UUID,Path> getSortedWALogs() {
+            return Collections.emptyMap();
+          }
+
+          @Override
+          WalStateManager createWalStateManager(ServerContext context) {
+            return marker;
+          }
+
+          @Override
+          Stream<TabletLocationState> createStore() {
+            return tabletOnServer2List;
```

Comment:
This one doesn't match the original gc object created with `tabletOnServer1List`. Is this intentional?

Reply:
Oops, no. Good catch.